persist-cli/src/main/java/io/ballerina/persist/cmd/Push.java

```diff
@@ -232,4 +252,27 @@ public void printLongDesc(StringBuilder out) {
     public void printUsage(StringBuilder stringBuilder) {
         stringBuilder.append("  ballerina " + COMPONENT_IDENTIFIER + " db push").append(System.lineSeparator());
     }
+
+    private Path getDriverPath() throws BalException {
+        String relativeLibPath;
+
+        DependencyGraph<ResolvedPackageDependency> resolvedPackageDependencyDependencyGraph =
+                balProject.currentPackage().getResolution().dependencyGraph();
+
+        ResolvedPackageDependency root = resolvedPackageDependencyDependencyGraph.getRoot();
+
+        Package mysql = resolvedPackageDependencyDependencyGraph.getDirectDependencies(root).stream().
+                filter(resolvedPackageDependency -> resolvedPackageDependency.packageInstance().
+                descriptor().toString().contains(BALLERINA_MYSQL_DRIVER_NAME)).findFirst().get().packageInstance();
+
+        List<Map<String, Object>> dependencies = mysql.manifest().platform(PLATFORM).dependencies();
+
+        for (Map<String, Object> dependency : dependencies) {
+            if (dependency.get(PATH).toString().contains(MYSQL_CONNECTOR_NAME_PREFIX)) {
+                relativeLibPath = dependency.get(PATH).toString();
+                return mysql.project().sourceRoot().resolve(relativeLibPath);
+            }
+        }
+        throw new BalException("Failed to retrieve MySQL driver path in the local cache");
```

Comment:
Shall we say how they can solve it?

Reply:
The MySQL driver jar would be missing in the local cache only if the user manually removes the driver import from the generated `database_configuration.bal` file. Perhaps we can ask the user to re-execute the `bal perist generate` command. But IMO this is an unlikely scenario.

Even if he deletes the ballerina cache, the MySQL driver jar will still be pulled from central and stored in the local cache since the project is being built during the command execution.

Reply:
Then shall we add the comment? // Unreachable code since this step is taken beforehand?

Reply:
Updated in [bf7200d](https://github.com/ballerina-platform/persist-tools/pull/53/commits/bf7200da5d01ea43bc5c86c115b324297a06bc57)